source/nodejs/adaptivecards-site/themes/adaptivecards/source/js/script.js

```diff
@@ -458,16 +458,14 @@ $(function () {
 		adaptiveCard.hostConfig = new AdaptiveCards.HostConfig(hostConfig);
 		var renderedCard;
 
-		var version = json.version.split(".");
-
 		if (dataJson) {
 			var template = new ACData.Template(json);
 			adaptiveCard.parse(template.expand({
 				$root: dataJson
-			}), new AdaptiveCards.SerializationContext(new AdaptiveCards.Version(version[0], version[1])));
+			}), new AdaptiveCards.SerializationContext(AdaptiveCards.Versions.Latest));
```

Comment:
What is the version specified in the sample? I think we parse for the version specified here.

Reply:
<img width="556" alt="image" src="https://github.com/microsoft/AdaptiveCards/assets/98650930/4bafad2a-0d7d-4473-a96b-5c109212d254">

We could go either way, but updating the version in the sample would fix this I think

Reply:
good point -- less scary fix too :)

Reply:
hmm... looking at this a bit more, we have a few samples that need updates for version...